lib/ruby_lsp/server.rb

```diff
@@ -55,6 +55,10 @@ module RubyLsp
         )
       end
 
+      inlay_hint_provider = if enabled_features.include?("inlayHint")
+        { resolveProvider: false }
```

Comment:
Now with the 3.17, we can use [inlay hint option](https://github.com/mtsmfm/language_server-protocol-ruby/blob/main/lib/language_server/protocol/interface/inlay_hint_options.rb) instead of the hash.

Reply:
Ooo good eye